tests/test_objective_funs.py

```diff
@@ -201,6 +202,26 @@ def test(eq):
         test(Equilibrium(iota=PowerSeriesProfile(0)))
         test(Equilibrium(current=PowerSeriesProfile(0)))
 
+    @pytest.mark.unit
+    def test_pressure(self):
+        """Test calculation of pressure objective."""
```

Comment:
I can try to answer here since this is a test case we reuse in this file, we take in an equilibrium `eq` into the testing function `test`, it uses that equilibrum to create a `Pressure` objective object `obj` here, with a target value of `1` 

We then make sure that the unscaled value of the objective `p` is the same as the pressure of the equilibrium we passed in (in this case is `12`), because the `compute_unscaled` method of an objective simply compute the value of the thing that we are targeting (in this case the pressure).

We also compute the error in satisfying the objective using `compute_scaled_error` which will not only compute the thing we are targeting, but then subtract from it our target value `1` to return the error in achieving that target value, which then is the thing that we try to minimize during our optimization. It also scales the error(by the `sqrt(3)` which I am unsure of what that is? @f0uriest can explain that to both of us) and multiplies by the objective's weight (`2` because we specified `weight=2` when creating the `Pressure` object)

Reply:
Thanks! So when the target of the pressure is 1. Is it one at a particular grid point or throughout the volume?

This test has too many numbers that seem to pop out of nowhere. This is hard to understand especially because each number has a different mean.

Reply:
in this case the target is 1 across the entire profile, but in general `target` can either be an array with the same size as the output (dependent on the `Grid` that is used by each objective), or for profile objectives like this, `target` can be a callable function ie `target(rho)`

The `sqrt(3)` factor are the quadrature weights (in this case its a linear grid with only 3 nodes)

Reply:
A comment like this: "Scaled target pressure is one throughout the volume. Test to check if it matches with the unscaled values of 12. weight = 2 and the sqrt(3) is a quadrature weight." should be useful.